src/main/java/org/mitre/stix/DocumentUtilities.java

```diff
@@ -35,21 +35,31 @@
 
 /**
  * A collection of utility helper methods.
- * 
+ *
  * @author nemonik (Michael Joseph Walsh <github.com@nemonik.com>)
  */
 public class DocumentUtilities {
 
 	private static final String XML_SCHEMA_INSTANCE = "http://www.w3.org/2001/XMLSchema-instance";
 	private static final String XML_NAMESPACE = "http://www.w3.org/2000/xmlns/";
 
+	private static final JAXBContext STIX_CONTEXT = initDefaultContext();
+
+	private static JAXBContext initDefaultContext(){
+		try {
+			return JAXBContext.newInstance("org.mitre.stix.stix_1");
```

Comment:
This JAXBContext instance seems very coupled to just the STIX_1 namespace.  I noticed that other generated binding classes use the same DocumentUtilitites.toDocument() method (i.e. types org.mitre.cybox.cybox_2).  

Will this JAXBContext still work in the case where the code is just calling the toDocument method on the cybox object? The cybox binding is under the package namespace that starts with "org.mitre" but is not "org.mitre.stix.stix_1".